mozilla/bedrock
# Add breach data model and sync command

Fixes #13611, #13612

## One-line summary

Add breach Django model and syncing management command.

## Issue / Bugzilla link

#13611 
#13612 

## Testing

To test locally:

1. (Optional) Pull down a fresh dev db. If you want to repeat this testing after having done these steps once, this will reset your database back to current "dev" state. This (or a `make preflight`) would also be advised to do when you're done testing this branch.

        `./bin/run-db-download.py --force`

2. Run the migration to create the breaches database table.
 
        `./manage.py migrate products`

3. Run the management command to sync the breaches, overriding the `STATIC_URL` environment variable since we need this to set the correct breach logo URLs. Normally this will run on our deployments on an interval and wouldn't be needed locally, but until this is merged we need to simulate a sync run as if we're on dev.

    This requires `gcloud` to be set up so we can upload to GCS. Make sure you're auth'd.

        `gcloud auth application-default login`

    Then run the sync:

        `STATIC_URL=https://www-dev.allizom.org/media/ ./manage.py sync_breaches`
